cmd/platformcli/cli.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/innosat-mats/level1a-platform/internal/platform"
+)
+
+var outputDirectory *string
+var stdout *bool
+
+type getter func(filename string) platform.Records
+
+func processFiles(
+	recordsGetter getter,
+	inputFiles []string, stdout bool, outputDirectory string) error {
+	for _, inputFile := range inputFiles {
+		records := recordsGetter(inputFile)
+		if stdout {
+			fmt.Println(records)
+		} else {
+			outputFile := platform.GetFilepath(
+				inputFile, outputDirectory)
+			err := records.Write(outputFile)
```

Comment:
It would be good if this is tested too I think

Reply:
it is tested

Reply:
I didn't see it in the unittests.

Reply:
vad jag menade var att detta felet är testat i cli_test "test write to disk fail returns error". kan lägga till ett unit test på det också.